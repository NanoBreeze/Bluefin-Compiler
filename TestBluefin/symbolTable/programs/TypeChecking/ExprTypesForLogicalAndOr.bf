bool a;
int main() {
	if (a && true) {}
	if (true || false) {}
}