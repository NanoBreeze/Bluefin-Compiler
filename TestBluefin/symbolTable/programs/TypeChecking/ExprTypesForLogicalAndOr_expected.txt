a - t_bool - t_bool
true - t_bool - t_bool
a&&true - t_bool - t_bool
true - t_bool - t_bool
false - t_bool - t_bool
true||false - t_bool - t_bool
